analytics/dsl/export/queries/locations.sql

```diff
@@ -0,0 +1 @@
+INSERT into locations SELECT t.*  from  analytics.analytics.locations t
```

Comment:
Recall we moved away from hard coding the catalog name in the query, use placeholder, see PR at https://github.com/ozone-his/ozone-analytics-queries/pull/8